### PR TITLE
added nodejs week2 homework

### PR DESCRIPTION
**Pull request by krithi-kondana**

### CODE REVIEW

File: nodejs/week2/meal_app/src/backend/routes/meals.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const app = express();
```
Comment (Hatko) [inline nodejs/week2/meal_app/src/backend/routes/meals.js]: Even though the code works, the object you use here should be a router (`express.Router()`)

File: nodejs/week2/meal_app/src/backend/routes/meals.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    meal.review = reviewsJson.filter(mealReview => mealReview.mealId === meal.id)
+});
+
+app.get("/", (req, res) => {
+    const maxPrice = req.query["maxPrice"];
+    const title = req.query["title"];
+    const createdAfter = req.query["createdAfter"];
+    const limit = req.query["limit"];
+
+    if (req.originalUrl === '/meals') {
```
Comment (Hatko) [inline nodejs/week2/meal_app/src/backend/routes/meals.js]: Please check my comment here:

https://github.com/Mekdes1/hyf-homework/pull/22#discussion_r387191005

File: nodejs/week2/meal_app/src/backend/routes/meals.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+const app = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+mealsJson.forEach(meal => {
+    meal.review = reviewsJson.filter(mealReview => mealReview.mealId === meal.id)
+});
+
+app.get("/", (req, res) => {
```
Comment (Hatko) [inline nodejs/week2/meal_app/src/backend/routes/meals.js]: Please check my comment here:

https://github.com/Mekdes1/hyf-homework/pull/22#discussion_r387193511

Comment (krithi-kondana) [reply nodejs/week2/meal_app/src/backend/routes/meals.js]: @hatko I am not able to crack this. Could you please tell how to do it?

**Pull request merged**